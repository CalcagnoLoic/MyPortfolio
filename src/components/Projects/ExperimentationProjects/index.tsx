const Component = () => {
  return (
    <>
      <h2
        id="experimentation"
        className="text-center font-[sofia] text-2xl md:text-4xl underline"
      >
        Experimentation projects
      </h2>
    </>
  );
};

export default Component;
