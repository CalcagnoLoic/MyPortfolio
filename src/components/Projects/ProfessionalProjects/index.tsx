const Component = () => {
  return (
    <>
      <h2
        id="projects"
        className="my-10 text-center font-[sofia] text-2xl underline md:text-4xl"
      >
        Recent professional projects
      </h2>
      <div></div>
    </>
  );
};

export default Component;
