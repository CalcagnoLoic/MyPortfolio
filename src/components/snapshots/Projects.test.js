import renderer from "react-test-renderer";
import Projects from "../Projects";

it("render correctly Projects component", () => {
    const componentProjects = renderer.create(<Projects />).toJSON();
    expect(componentProjects).toMatchSnapshot();
});
