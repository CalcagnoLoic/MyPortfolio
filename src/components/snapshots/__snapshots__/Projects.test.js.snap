// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Projects component 1`] = `
<section
  className="px-5 sm:px-5 md:px-8 lg:px-28 pb-20 pt-20"
>
  <h2
    className="text-center text-main-color text-3xl lg:text-4xl font-bold font-[sofia] underline pb-5"
    id="projects"
  >
    Recent projects
  </h2>
  <div
    className="grid grid-cols-1 sm:grid-cols-2 xl:grid-cols-3 sm:gap-10"
  >
    <div
      className="group flex flex-col gap-5 md:gap-0 lg:gap-0 justify-center mt-7 rounded-xl overflow-hidden shadow-slate-700/50 shadow-lg"
    >
      <a
        href="https://calcagnoloic.github.io/Sunnyside-agency-react/"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Projet"
          className="w-full cursor-pointer rounded-xl hover:scale-125 transition-all duration-700 hover:opacity-30"
          src="./img/sunnyside.png"
        />
      </a>
      <div
        className="absolute"
      >
        <div
          className="px-14 py-3 rounded-r-full bg-transparent group-hover:bg-slate-400 duration-700"
        >
          <a
            href="https://github.com/CalcagnoLoic/Sunnyside-agency-react/tree/main"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Logo Github"
              className="w-16 opacity-0 group-hover:opacity-100 duration-700"
              src="./img/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="group flex flex-col gap-5 md:gap-0 lg:gap-0 justify-center mt-7 rounded-xl overflow-hidden shadow-slate-700/50 shadow-lg"
    >
      <a
        href="https://calcagnoloic.github.io/loopstudios/"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Projet"
          className="w-full cursor-pointer rounded-xl hover:scale-125 transition-all duration-700 hover:opacity-30"
          src="./img/loopstudio.png"
        />
      </a>
      <div
        className="absolute"
      >
        <div
          className="px-14 py-3 rounded-r-full bg-transparent group-hover:bg-slate-400 duration-700"
        >
          <a
            href="https://github.com/CalcagnoLoic/loopstudios"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Logo Github"
              className="w-16 opacity-0 group-hover:opacity-100 duration-700"
              src="./img/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="group flex flex-col gap-5 md:gap-0 lg:gap-0 justify-center mt-7 rounded-xl overflow-hidden shadow-slate-700/50 shadow-lg"
    >
      <a
        href="https://space-tourism-loic.netlify.app/"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Projet"
          className="w-full cursor-pointer rounded-xl hover:scale-125 transition-all duration-700 hover:opacity-30"
          src="./img/spacetourism.png"
        />
      </a>
      <div
        className="absolute"
      >
        <div
          className="px-14 py-3 rounded-r-full bg-transparent group-hover:bg-slate-400 duration-700"
        >
          <a
            href="https://github.com/CalcagnoLoic/space-tourism"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Logo Github"
              className="w-16 opacity-0 group-hover:opacity-100 duration-700"
              src="./img/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</section>
`;
